test/e2e/tests/tokens/custom-token-send-transfer.spec.js

```diff
@@ -147,7 +147,7 @@ describe('Transfer custom tokens @no-mmi', function () {
         const tokenAmount = await driver.findElement(
           {
             css: '[data-testid="multichain-token-list-item-value"]',
-            text: '8.5 TST',
+            text: '8.5000 TST',
```

Comment:
I have a more legitimate fix for this incoming. The balance not updating immediately has been fixed, but we need to correctly format this value.